docs/en/operations/utils/clickhouse-benchmark.md

```diff
@@ -82,64 +81,66 @@ In the report you can find:
 
     - Endpoint of ClickHouse server.
     - Number of processed queries.
-    - QPS. Queries Per Second server metric for the period (`--delay`) of reporting.
-    - RPS. Rows Per Second server metric for the period (`--delay`) of reporting.
-    - MiB/s. Megabytes Per Second server metric for the period (`--delay`) of reporting.
-    - result RPS. Rows Per Second server metric for the whole period.
-    - result MiB/s. Megabytes Per Second server metric for the whole period.
+    - QPS: Metric "how many queries server performed per second" that measured in a period (`--delay`) of reporting.
```

Comment:
word "metric" and double quotes are kind of useless here

could be just something like "QPS: how many queries server performed per second during a period specified in `--delay` argument"